ModAssistant/Localisation/sv.xaml

```diff
@@ -0,0 +1,232 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="clr-namespace:ModAssistant"
+    xmlns:sys="clr-namespace:System;assembly=mscorlib">
+    <sys:String x:Key="ResourceDictionaryName">i18n:en-US</sys:String>
+
+    <!-- App -->
+    <sys:String x:Key="App:InstallDirDialog:Title">Det gick inte att hitta din Beat Saber-installationsmapp!</sys:String>
+    <sys:String x:Key="App:InstallDirDialog:OkCancel">Tryck på OK för att försöka igen, eller Cancel för att avsluta applikationen.</sys:String>
+    <sys:String x:Key="App:InvalidArgument">Invalid argument! '{0}' requires an option.</sys:String>
+    <sys:String x:Key="App:UnrecognizedArgument">Unrecognized argument. Stänger Mod Assistant.</sys:String>
+    <sys:String x:Key="App:UnhandledException">An unhandled exception just occurred</sys:String>
+    <sys:String x:Key="App:Exception">Undantag</sys:String>
+
+    <!-- Main Window -->
+    <sys:String x:Key="MainWindow:WindowTitle">ModAssistant</sys:String>
+    <sys:String x:Key="MainWindow:IntroButton">Intro</sys:String>
+    <sys:String x:Key="MainWindow:ModsButton">Mods</sys:String>
+    <sys:String x:Key="MainWindow:AboutButton">Om</sys:String>
+    <sys:String x:Key="MainWindow:OptionsButton">Inställningar</sys:String>
+    <sys:String x:Key="MainWindow:GameVersionLabel">Spelversion</sys:String>
+    <sys:String x:Key="MainWindow:VersionLabel">Version</sys:String>
+    <sys:String x:Key="MainWindow:ModInfoButton">Modinformation</sys:String>
```

Comment:
This appears to be untranslated. Is this right?

Reply:
Yes! This is correct! The only difference between English and Swedish here is essentially the fact that "Mod Information" is "Modinformation" in Swedish.

Reply:
Ok, no problem 👍